.github/workflows/check-ndi-sdk.yaml

```diff
@@ -0,0 +1,30 @@
+name: Check for updates
+on:
+  schedule:
+  - cron: "* * * * 1"
+  workflow_dispatch: 
+
+jobs:
+  flatpak-external-data-checker:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        branch: [ master ]
```

Comment:
if there is only one branch why is this still needed? You can use the global x-checker?

Reply:
It was from the example itself.